conjur-api/Resource.cs

```diff
@@ -110,5 +110,7 @@ public bool Check(string privilege)
                 offset += (uint)resultList.Count;
             } while (resultList.Count > 0);
         }
+
+        protected static string IdToName (string id, string account, ResourceKind kind) => id.Substring (id.IndexOf ($"{account}:{kind}:") + 1);
```

Comment:
Is there any reason not to just pass the id and return the last part? 
Also, this function would be clearer using full function definition syntax.